### DISCUSSION TITLE
What is the goal of mini.nvim?

### DISCUSSION THREAD

**Discussion by tydra-wang** [nvim-mini/mini.nvim#143] [Q&A] [answered]
I am wondering the goal of mini.nvim. Rewrite those "similar plugins" and replace them?

**Comment by echasnovski** [comment, marked as answer]
There are at least three reasons for me thus far:
- Enjoy writing new modules.
- Return back to Neovim community with new functionality and ideas.
- Eventually own most of my not language specific (like 'nvim-treesitter', 'nvim-web-devicons', etc.) Neovim setup.
